### ISSUE TITLE
Correct descriptions of WebSocket API CloudWatch metrics

### ISSUE THREAD

**Issue by rwalle61** [awsdocs/amazon-api-gateway-developer-guide#136]
Hi, thanks for the developer guide, it's helped me out a lot! 😄 

The metric descriptions on the [Monitoring WebSocket API execution with CloudWatch metrics](https://github.com/awsdocs/amazon-api-gateway-developer-guide/blob/main/doc_source/apigateway-websocket-api-logging.md) page seem a bit inaccurate or insufficient. It'd be great to clarify the `ClientError`, `IntegrationError` and `ExecutionError` metric descriptions.

`ClientError` is supposedly triggered _before_ the integration is invoked:
![image](https://user-images.githubusercontent.com/18170169/137299897-9fd9c839-5ccc-4ef5-b604-9f33f1b4e4a3.png)

But I can trigger it _after_ the integration is invoked. For example, if I connect my WebSocket API to a lambda proxy integration, and make that handler return `{ statusCode: 400 }`, then the `ClientError` metric increases:
![image](https://user-images.githubusercontent.com/18170169/137303650-2953d1f5-87ba-426d-9e80-4f2e4905a9ea.png)

The current description seems misleading, suggesting that `ClientError` only happens before the integration is invoked - i.e. due to authoriser errors or [request validation](https://docs.aws.amazon.com/apigateway/latest/developerguide/websocket-api-request-validation.html).

Also, it'd be great if a doc explained how the WebSocket API, API Gateway service, stage, integration, and lambda actually interact. For example, it's not clear why:
* the lambda integration always return 200, even if my lambda handler returns other `statusCode`
* if I misconfigure the integration's permissions, the integration request fails, and `ExecutionError` metric increases but not `IntegrationError`:
![image](https://user-images.githubusercontent.com/18170169/137303271-3062f30f-140c-427f-a21a-9a6620c9d373.png)

In summary, I'd really appreciate your clarification on what the `ClientError`, `IntegrationError` and `ExecutionError` metrics really represent for WebSocket and REST APIs. Thanks 🙂

**Comment by joshbean** [comment]
Closing this issue or pull request in advance of archiving this repo. For more information about the decision to archive this repo (and others in the 'awsdocs' org), see [the announcement on the AWS News Blog](https://aws.amazon.com/blogs/aws/retiring-the-aws-documentation-on-github/).
